lab_solutions/lab02_08_get_corpus_word_frequencies.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Attila Nagy <attila.nagy234@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+from lab_solutions.lab02_05_word_freq import word_freq
+from lab_solutions.lab02_06_build_wikipedia_corpus import build_wikipedia_corpus
+
+
+def get_and_pretty_print_top_k(freq_dict, k=10):
+    list_to_print = sorted(list(freq_dict.items()), key=lambda x: x[1], reverse=True)[:k]
+    for key, value in list_to_print:
+        print(key + '\t' + str(value))
+
+    return list_to_print
```

Comment:
It's not a good idea to mix printing and returning in the same function since print is a side effect. If a function prints something, that should be its sole responsibility.